views/dailystatus.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Space - Daily Status</title>
+	<script type="text/javascript" src="../assets/js/lib/angular.min.js"></script>
```

Comment:
Why did you refer anular.js here? This should be referred in main index file rt?